client/.eslintrc

```diff
@@ -18,6 +18,7 @@
    "react/react-in-jsx-scope": "off",
    "react/prop-types": "off",
    "react-hooks/rules-of-hooks": "error",
-   "react-hooks/exhaustive-deps": "error"
+   "react-hooks/exhaustive-deps": "error",
+   "prettier/prettier": 0
```

Comment:
Can you explain the decision behind this? Turning off prettier will only make the project inconsistent.

Reply:
I would like you to not disable Prettier as it will lead to inconsistencies in the project as everyone has their own way of programming. If you are working off of VS Code, take a look at the [Prettier extension](https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode) and also in VS Code you can set `formatOnSave` which will automatically run the Prettier formatter and make sure your code is compliant with the linter.